lib/shopify_app/middleware/jwt_middleware.rb

```diff
@@ -0,0 +1,41 @@
+module ShopifyApp
+  class JWTMiddleware
+    TOKEN_REGEX = /^Bearer\s+(.*?)$/
+
+    def initialize(app)
+      @app = app
+    end
+
+    def call(env)
+      return call_next(env) unless authorization_header?(env)
+
+      token = extract_token(env)
+      return call_next(env) unless token
+
+      set_env_variables(token, env)
+      call_next(env)
+    end
+
+    private
+
+    def call_next(env)
+      @app.call(env)
+    end
+
+    def authorization_header?(env)
+      env['HTTP_AUTHORIZATION']
+    end
```

Comment:
Should this be explicitly returning true/false? (or drop the `?` from the method name?)